tflitehub/mobilebert_edgetpu_s_float_test.py

```diff
@@ -0,0 +1,34 @@
+# RUN: %PYTHON %s
+
+import absl.testing
+import numpy
+import test_util
+
+model_path = "https://storage.googleapis.com/iree-model-artifacts/mobilebert-edgetpu-s-float.tflite"
```

Comment:
nit: Since the models are from TFHUB, I wonder if we can pull directly from there, similar to 
https://github.com/google/iree-samples/blob/main/tflitehub/mobilenet_quant_test.py#L10

Reply:
It looks like only the quantized version of MobileBERT-EdgeTPU is available on TFHUB. I contacted the author for the other versions. I'd like to make sure the float and quantized versions of the same model don't move out of sync, so would prefer to reference all models from the single IREE storage location. What do you think?

Reply:
That's fine. It is a nit commit to begin with. My only concern is that tfhub may update the model, but here a snapshot of it is tested.